frontend/src/container/GridGraphLayout/Graph/index.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/mouse-events-have-key-events */
```

Comment:
Can you remove this eslint disable rule and attach a keyboard listener

Reply:
Done